src/textual/widgets/_tabbed_content.py

```diff
@@ -422,38 +534,77 @@ def _is_associated_tabs(self, tabs: Tabs) -> bool:
         """
         return isinstance(tabs, ContentTabs) and tabs.tabbed_content is self
 
-    def _watch_active(self, active: str) -> None:
+    def _watch_active(self, old_active: str, active: str) -> None:
```

Comment:
You're not really using this `old_active`, are you?

Reply:
Doesn't seem so; I would suspect it being vestigial of prior WiP.